src/domain.ts

```diff
@@ -62,13 +62,30 @@ interface DocMetaData {
   character: number;
 }
 
-interface TestMetaData {
+interface TestMetadataKwArgs {
```

Comment:
I think the naming can be better, for example `TestSpecification`

Reply:
fixed